components/ReviewModal.js

```diff
@@ -1,24 +1,78 @@
+import React from 'react';
+import axios from 'axios';
 import Button from 'react-bootstrap/Button';
 import Modal from 'react-bootstrap/Modal';
 import ReviewModalForm from '../components/ReviewModalForm.js'
 
-export default function ReviewModal({ show, handleClose }) {
-  return (
-    <Modal show={show} onHide={handleClose} centered>
-    <Modal.Header closeButton>
-      <Modal.Title>Add A New Review</Modal.Title>
-    </Modal.Header>
-    <Modal.Body>
-      <ReviewModalForm />
-    </Modal.Body>
-    <Modal.Footer>
-      <Button variant="secondary" onClick={handleClose}>
-        Close
-      </Button>
-      <Button variant="success" onClick={handleClose}>
-        Submit Review
-      </Button>
-    </Modal.Footer>
-  </Modal>
-  )
+export default class ReviewModal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      user: '',
+      workDate: '',
+      body: '',
+      title: '',
+      img: [],
+      categories: [],
+      city: '',
+      state: ''
+    }
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.handleCheck = this.handleCheck.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleInputChange = (e) => {
+    e.preventDefault();
+    const target = e.target;
+    const value = target.type === 'checkbox' ? target.checked : target.value;
+    const name = target.name;
+
+    this.setState({
+      [name]: value,
+    });
+  }
+  handleCheck = (e) => {
+    let temp = this.state.categories;
+    if (temp.indexOf(e.target.name) !== -1) {
+      temp.splice(temp.indexOf(e.target.name), 1);
+      this.setState({
+        categories: temp
+      });
+    } else {
+      temp.push(e.target.name);
+      this.setState({
+        categories: temp
+      })
+    }
+  }
+  handleSubmit = () => {
+    axios.post('http://localhost:8080/addreview', this.state)
```

Comment:
Did you test these routes with the database already?

Reply:
Yes! Jacob had helped me with making the routes, so we crosschecked with the database to make sure data was able to be successfully sent/received.